cis.py

```diff
@@ -25,9 +22,7 @@ def run(command, env={}):
                                env=merged_env)
     while True:
         line = process.stdout.readline()
-        #line = str(line).encode('utf-8')[:-1]
-        line=str(line, 'utf-8')[:-1]
-        print(line)
```

Comment:
Will removing this print statement also prevent the job from writing to the output file?

Reply:
Good point. I don't know, we can leave it in. It also ends up filling the output file with a ton of blank lines for some reason.

Reply:
I put it back.

Reply:
can we just set `line = line.decode('utf-8')[:-1]`? It's more explicit that it's recoding from a byte string

Reply:
I replaced it.